vic/extensions/rout_irr/src/rout_init.c

```diff
@@ -0,0 +1,224 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+#include <vic_def.h>
+#include <vic_run.h>
+#include <vic_driver_image.h>
+#include <netcdf.h>
+#include <rout.h>
+
+void rout_init(void){
+    extern rout_struct rout;
+    
+    size_t x;
+    size_t y;
+    size_t t;
+    
+    for (y=0;y<rout.y_size;y++){
+        for(x=0;x<rout.x_size;x++){
+            rout.cells[y][x].x=x;
+            rout.cells[y][x].y=y;
+            
+            if(rout.cells[y][x].nr_upstream==-1){
+                rout.cells[y][x].active=0;
+            }else{
+                rout.cells[y][x].active=1;
+            }
+            
+            for(t=0;t<rout.total_time;t++){
+                rout.cells[y][x].outflow[t]=0.0;
+            }
+            
+            /*for (ui = 0; ui < local_domain.ncells_active; ui++) {
+                if (lat[x][y] == local_domain.locations[ui].latitude &&
+                    lon[x][y] == local_domain.locations[ui].longitude) {
+                    rout.cells[x][y].vic_id = ui;
+                }
+            }*/
+        }
+    }
+        
+    set_upstream("/home/bram/VIC5.0Routing/VICTestSetup/input/VIC_Params/RVIC_input_NL.nc","flow_direction");
```

Comment:
het zou op zich fijn zijn als je de paden op z'n minst relatief maakt (dan kan ik het ook testen zonder teveel in jou code te klooien)